blockchain/fullblocktests/generate.go

```diff
@@ -2168,7 +2168,7 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	testInstances = nil
 	reorgSpend = *outs[spendableOutOffset]
 	reorgTicketSpends = ticketOuts[spendableOutOffset]
-	chain2TipName := g.TipName()
+	var chain2TipName string
```

Comment:
I would prefer this stay as it was since the this change would lead to a false positive fail if `numLargeReorgBlocks` were ever changed to 0 during debugging.

Reply:
it is either set immediately in the for loop, or then set below with  chain2TipName = g.TipName()

Reply:
i'd like to leave ineffassign on since it has found actual bugs.